Assets/Scripts/Augment/AugmentImplementations/PanHitbox.cs

```diff
@@ -0,0 +1,20 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PanHitbox : HitboxController
+{
+    [SerializeField]
+    private HealthController panHealth;
```

Comment:
Superfluous. The superclass already has a public health member.

Reply:
They have two different purposes, the panhealth is for the panshotemitter, not for tracking actual damage!

Reply:
Oh okay